lib/stripe.js

```diff
@@ -121,9 +117,6 @@ const resources = {
   }),
 };
 
-// For backwards compatibility, `Files` is aliased to `FileUploads`
-resources.FileUploads = resources.Files;
```

Comment:
So we're sure we want to remove this? I think that's what we gavel-ed but wanted to reflag so that the IRL discussion is tracked in text

Reply:
I'm +1 to removing `FileUploads`.

Reply:
👍